packages/nocodb/src/lib/meta/api/userApi/initStrategies.ts

```diff
@@ -103,8 +103,8 @@ export function initStrategies(router): void {
 
         if (cachedVal) {
           if (
-            cachedVal.token_version &&
-            jwtPayload.token_version &&
+            !cachedVal.token_version ||
```

Comment:
@mertmit Mind explain the changes on this part? 

This may cause a problem.

1. Login as admin
2. Change password
3. Password is changed -> logged out automatically
4. Login as admin
5. It throws `Token Expired. Please login again.` which is not expected.
6. Login again - it works 

I remember one of them will be set after this function. Hence, I didn't check both `token_version` for this case. Otherwise, it'd get the above problem.

Reply:
Former state allow token validation in case of null token from cache or jwt payload. The problem you mentioned looks like can be fixed using meta as dropback (when cache fails) but real problem here looks like even the signIn request success why jwt not updated for first login. (signin request success, token returned but token doesn't work for further requests)
Requests:
![image](https://user-images.githubusercontent.com/59797957/175771867-11442a76-14f8-4b95-816d-d08b99031cd1.png)

/me response after first successful signin request(looks like return successful but fails actually causing the issue):
![image](https://user-images.githubusercontent.com/59797957/175771887-d46d2b10-a43f-44ae-b85b-811ac7e6f6c7.png)

/me response after second signin:
![image](https://user-images.githubusercontent.com/59797957/175771905-6ec25389-8813-48b3-9d7c-e3929c85d39f.png)

I couldn't find a fix other than in case of failed auth drop back into meta. How we should handle this?

Reply:
Fixed the issue in the latest commit. @wingkwong @mertmit Please test it.



https://user-images.githubusercontent.com/61551451/175930129-8957d6a1-2a3f-4c1d-a1b8-e9f20df0cd2d.mov